application/app/src/main/java/com/example/liftvectr/activities/AddExerciseActivity.java

```diff
@@ -54,7 +53,7 @@ public class AddExerciseActivity extends AppCompatActivity {
     // For emulating ONLY
     // MODIFY this to true to allow start/stop exercise to be pressed, creating a fake exercise and transitioning
     // to CropExerciseActivity during emulation
-    private boolean emulationMode = false;
+    private boolean emulationMode = true; // TODO: change back to false
```

Comment:
Let's change this back to false. Need to make sure our main branch always has bluetooth enabled.